### PR TITLE
JBPM-6120 - Invalid Kie Server REST endpoints for processes

### PR DESCRIPTION
**Pull request by jakubschwan**
Fixed REST endpoints name.
Added missing tests scenarios.

### CODE REVIEW

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by jakubschwan** [comment]
add fix for org.kie.server.integrationtests.jbpm.ProcessServiceIntegrationTest.testFindVariableInstances

**Comment by jakubschwan** [comment]
@mswiderski We can get process instances from process uri REST endpoint `server/containers/{id}/processes/processes/instances - GET` but it is not possible to get process instance from ProcessServicesClient. We should add this functionality into client. WDYT?

**Comment by mswiderski** [comment]
@jakubschwan in general this endpoint is mainly a short cut for non java clients as it is more intuitive to get it like that. ProcessServicesClient has not be enhanced to make use of this yet as same feature is available in QueryServicesClient. But when you brought this up maybe it is a good idea to actually add it there too.

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by tsurdilo** [comment]
@jakubschwan jenkins fail report:
******
org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testGetProcessInstances[JSON]
Error Message

org.codehaus.jackson.map.JsonMappingException: Can not deserialize instance of java.util.Date out of START_OBJECT token
 at [Source: org.apache.http.conn.EofSensorInputStream@78b850b2; line: 9, column: 25] (through reference chain: org.kie.server.api.model.instance.ProcessInstanceList["process-instance"]->org.kie.server.api.model.instance.ProcessInstance["start-date"])
******

seems added test is failing (on line 129)

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/ProcessServiceRestOnlyIntegrationTest.java [hunk 1]
```diff
@@ -94,4 +98,72 @@ public void testAbortAlreadyAbortedProcess() {
...
+            assertNotNull(resultOne);
+
+            // get processes instances
+            WebTarget clientGetRequest = newRequest(build(TestConfig.getKieServerHttpUrl(), PROCESS_URI + "/" + PROCESS_INSTANCES_BY_CONTAINER_GET_URI, valuesGetMap));
+            logger.debug("[GET]" + clientGetRequest.getUri());
+
+            response = clientGetRequest.request(getMediaType()).get();
+            Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+
+            List<ProcessInstance> resultList = response.readEntity(ProcessInstanceList.class).getItems();
```
Comment (tsurdilo) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/ProcessServiceRestOnlyIntegrationTest.java]: test fails on jenkins here with "javax.ws.rs.ProcessingException: org.codehaus.jackson.map.JsonMappingException: Can not deserialize instance of java.util.Date out of START_OBJECT token"

**Comment by jakubschwan** [comment]
@mswiderski I just added method findProcessInstances into ProcessServicesClient
@tsurdilo The failing test was replaced by tests for findProcessInstances

**Comment by mswiderski** [comment]
@jakubschwan thanks a lot for adding these methods to ProcessServicesClient

**Pull request merged**